src/rewriter.coffee

```diff
@@ -131,6 +141,18 @@ class exports.Rewriter
       i -= 1
     @tag(i) in tags
 
+  # Adjust token stream so that CoffeeTags look like normal function calls.
+  adjustCoffeeTags: ->
+    @scanTokens (token, i, tokens) ->
+      if token[0] is 'CT'
+        f = generate 'IDENTIFIER', 'h'; f.spaced = true
```

Comment:
We can’t merge in this PR if what it does is replace this new coffee tag literal with calls to a mysterious `h` function that is not defined within CoffeeScript. That makes coffee tags dependant on some external library to supply this `h` function in the global scope.